package/yast2-network.changes

```diff
@@ -1,10 +1,19 @@
 -------------------------------------------------------------------
+Wed Sep 21 11:56:38 UTC 2016 - kanderssen@suse.com
+
+- Bridge handling has been improved (bsc#962824).
+  - "NONE" is shown instead of 0.0.0.0 for old bridge configuration
+  - The bridge master is shown in the enslaved interface.
+  - The interfaces overview is updated after a bridge is modified
+- 3.1.170.2
+
+-------------------------------------------------------------------
 Mon Sep 12 09:40:57 UTC 2016 - kanderssen@suse.com
 
-- Use full interface up / down instead of setting just link 
+- Use full interface up / down instead of setting just link
   up / down to allow proper update of interface's configuration.
   (bsc#991694)
-- 3.1.171
+- 3.1.170.1
```

Comment:
in my POV you don't need the fourth number as we decided that SP3 will use 3.2.x format